GUI/Controls/EditModSearch.cs

```diff
@@ -263,9 +262,10 @@ private void FilterTextBox_KeyDown(object sender, KeyEventArgs e)
             }
         }
 
-        private void TriggerSearchOrTimer()
+        private void QueueTriggerSearch()
         {
-            if (Platform.IsMono && !string.IsNullOrEmpty(FilterCombinedTextBox.Text))
+            filterTimer?.Stop();
+            if (!string.IsNullOrEmpty(FilterCombinedTextBox.Text))
             {
                 // Delay updating to improve typing performance on OS X and Linux
```

Comment:
We should probably remove the explicit mention of "OS X and Linux" now that it is the case for all OSes.